### DISCUSSION TITLE
BEV Grundstücke Plugin

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#152] [General]
Hallo Jürgen!

In Niederösterreich gibt es Katastralgemeinden mit einer 4-stelligen KG-Nummer. Diese KG-Nummer ist bei uns in den Daten auch als 4-stellige Nummer hinterlegt (z.B. 9135 -  Ziersdorf).
Beim Absprung vom WebGIS zum BEV-Kataster geben wir die KG-Nummer deshalb auch als 4-stellige Zahl mit. Der Kataster vom BEV benötigt allerdings für die Abfrage eine 5-stellige KG-Nummer mit führender Null (09135). Sonst kommt eine Fehlermeldung ("Falsche Katastralgemeinde Nummer"). 

Test-URL (OK) --> https://kataster.bev.gv.at/api/gst/09135/1
Test-URL (NOK) --> https://kataster.bev.gv.at/api/gst/9135/1

Besteht die Möglichkeit in dem Plugin die KG-Nummer automatisch als 5-stellige Zahl zu definieren? Falls die KG-Nummer 4-stellig ist, soll automatisch eine führende Null hinzugefügt werden. 

Und mir sind noch zwei Rechtschreibfehler aufgefallen:
In der component.js in Zeile 93 steht "Rechtstatus" statt "Recht**ss**tatus" (Doppel s)
Und in Zeile 117 steht "Er**s**tragsmesszahl" statt "Ertragsmesszahl" (-s).

Lg, Andi

**Comment by gis-eni** [comment]
Danke, für den Hinweis, ist erledigt. Mit dem dann nächsten Release dann einfach die ``bev_gst/components.js`` austauschen.

**Comment by Andreas-Lampert** [comment]
Funktioniert. Danke Jürgen!
